assets/messages.yml

```
@@ -761,15 +928,32 @@ messages:
         - mcpe
         - realms
       name: Parity issue
+      shortcut: parity
       message: |-
-        Minecraft has several different editions, and some features vary from edition to edition. Often this happens when one edition evolves faster than another, and tweaks or balances are made that don't propagate immediately to other editions.
-        Parity issues are not tracked on the bug tracker but should instead be reported on the [Feedback website|https://feedback.minecraft.net/hc/en-us/community/topics/360001191251-Parity].
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        You have posted a parity issue. Minecraft has several different editions, and some features vary from edition to edition.
+        This is especially the case with features that were added a long time ago, because one edition evolved faster than another, and tweaks or balances were made that did not propagate immediately to other editions.
+        These differences between the different versions of the game are called "Parity issues".
+
+        However, we only accept bug reports about a parity issue if it fulfills the following criteria:
```

```suggestion
        We only accept bug reports about a parity issue if it fulfills the following criteria:
```
This does not conflict with the previous paragraph so "However" seems to not be necessary.